### DISCUSSION TITLE
CPU optimization preferences

### DISCUSSION THREAD

**Discussion by RinCat** [mastodon/mastodon#17117] [Q&A]
High CPU cores (8C 16T) but low clock speed (2.5 GHz)
VS
Low CPU cores (4C 8T) but high clock speed (3.8 GHz+)

Which one could works better? Assume all other hardware is identical.

Thanks.

**Comment by ClearlyClaire** [comment]
The high CPU cores one might be better, but I'm not sure, and I don't think there would be a big difference anyway. This may also slightly depend on how much RAM you have, e.g. if you're RAM-constrained you may want to limit the number of running workers, and thus would benefit more from fewer, faster cores, than more, slower cores.
